src/main/resources/assets/chisel/lang/de_de.lang

```diff
@@ -0,0 +1,2774 @@
+tile.chisel.block_coal.twisted.desc.1=Verdreht
+tile.chisel.block_coal.prism.desc.1=Prisma
+tile.chisel.block_coal.raw.desc.1=Roh
+tile.chisel.block_coal.emboss.desc.1=Geprägt
+tile.chisel.block_coal.indent.desc.1=Eingerückt
+tile.chisel.block_coal.marker.desc.1=Zeichen
+tile.chisel.block_coal.sunken.desc.1=Versunken
+tile.chisel.block_coal.poison.desc.1=Gift
+tile.chisel.block_coal.bricks-chaotic.desc.1=Zertretene Ziegel
+tile.chisel.block_coal.cuts.desc.1=Schnitte
+
+tile.chisel.block_coal_coke.name=Kohlekoksblock
+tile.chisel.block_coal_coke.cracked.desc.1=Gebrochen
+tile.chisel.block_coal_coke.bricks-soft.desc.1=Verwitterte Ziegel
+tile.chisel.block_coal_coke.bricks-cracked.desc.1=Gebrochene Ziegel
+tile.chisel.block_coal_coke.bricks-triple.desc.1=Breite Ziegel
+tile.chisel.block_coal_coke.bricks-encased.desc.1=Umhüllte Ziegel
+tile.chisel.block_coal_coke.braid.desc.1=Verflechtet
+tile.chisel.block_coal_coke.array.desc.1=Aufgestellte Ziegel
+tile.chisel.block_coal_coke.tiles-large.desc.1=Große Fliesen
+tile.chisel.block_coal_coke.tiles-small.desc.1=Kleine Fließen
+tile.chisel.block_coal_coke.chaotic-medium.desc.1=Ungeordnete Fließen
+tile.chisel.block_coal_coke.chaotic-small.desc.1=Kleine Ungeordnete Fließen
+tile.chisel.block_coal_coke.dent.desc.1=Vertiefung
+tile.chisel.block_coal_coke.french-1.desc.1=Französisch 1
+tile.chisel.block_coal_coke.french-2.desc.1=Französisch 2
+tile.chisel.block_coal_coke.jellybean.desc.1=Angeordnet
+tile.chisel.block_coal_coke.layers.desc.1=Schichten
+tile.chisel.block_coal_coke.mosaic.desc.1=Mosaik
+tile.chisel.block_coal_coke.ornate.desc.1=Kunstvoll
+tile.chisel.block_coal_coke.panel.desc.1=Vertäfelt
+tile.chisel.block_coal_coke.road.desc.1=Straße
+tile.chisel.block_coal_coke.slanted.desc.1=Geneigt
+tile.chisel.block_coal_coke.zag.desc.1=Zackig
+tile.chisel.block_coal_coke.circularct.desc.1=Kreisförmig
+tile.chisel.block_coal_coke.circularct.desc.2=Hat CTM
+tile.chisel.block_coal_coke.weaver.desc.1=Keltisch
+tile.chisel.block_coal_coke.bricks-solid.desc.1=Ziegel
+tile.chisel.block_coal_coke.bricks-small.desc.1=Kleine Ziegel
+tile.chisel.block_coal_coke.circular.desc.1=Kreisförmig
+tile.chisel.block_coal_coke.tiles-medium.desc.1=Fliesen
+tile.chisel.block_coal_coke.pillar.desc.1=Säule
+tile.chisel.block_coal_coke.twisted.desc.1=Verdreht
+tile.chisel.block_coal_coke.prism.desc.1=Prisma
+tile.chisel.block_coal_coke.raw.desc.1=Roh
+tile.chisel.block_coal_coke.emboss.desc.1=Geprägt
+tile.chisel.block_coal_coke.indent.desc.1=Eingerückt
+tile.chisel.block_coal_coke.marker.desc.1=Zeichen
+tile.chisel.block_coal_coke.sunken.desc.1=Versunken
+tile.chisel.block_coal_coke.poison.desc.1=Gift
+tile.chisel.block_coal_coke.bricks-chaotic.desc.1=Zertretene Ziegel
+tile.chisel.block_coal_coke.cuts.desc.1=Schnitte
+
+tile.chisel.bloodMagic.name=Gemeißelte Blutrune
+tile.chisel.bloodMagic.bloodRuneArranged.desc.1=Angeordnet
+tile.chisel.bloodMagic.bloodRuneArranged.desc.2=Safe zur Dekoration
```

Comment:
okay. I'll change it but considering that this will be displayed when hovering the block/item I would say that it's a bit to long. We'll see how it works out.